.github/workflows/main.yaml

```diff
@@ -171,22 +181,31 @@ jobs:
       - name: Test
         run: |
           make test
-        if: matrix.target != 'aarch64-apple-darwin'
+        if: matrix.target != 'aarch64-apple-darwin' && matrix.build != 'linux-musl-x64'
+      - name: Test (musl)
+        if: matrix.build == 'linux-musl-x64'
+        run: |
+          LIBC=musl WASMER_CAPI_USE_SYSTEM_LIBFFI=1 make test
```

Comment:
I wonder if it's possible to have this be one step by using the matrix to set the env vars ahead of time...

That would be optimal but I find it easy to believe that that's something that sounds way easier than it actually is to do.